daemon/containerd/image_delete.go

```diff
@@ -411,6 +385,25 @@ func (*imageDeleteConflict) Conflict() {}
 // nil if there are none. It takes a bitmask representing a
 // filter for which conflict types the caller cares about,
 // and will only check for these conflict types.
+
+// untagReferences deletes the given image references and returns the appropriate response records
+func (i *ImageService) untagReferences(ctx context.Context, refs []c8dimages.Image) ([]imagetypes.DeleteResponse, error) {
```

Comment:
Looks like the GoDoc for `checkImageDeleteConflict` got separated from the function

Reply:
whoops, thanks!